contracts/Everlong.sol

```diff
@@ -281,17 +284,21 @@ contract Everlong is IEverlong {
             revert IEverlong.RedemptionZeroOutput();
         }
 
-        // If we have enough balance to service the withdrawal after closing
-        // any matured positions, there's no need to close immature positions.
+        // TODO: Determine if we want to close enough positions to leave the
```

Comment:
@jalextowle Curious as to your thoughts on this

Reply:
I don't have a strong preference one way or another. It feels like this is decreasing gas costs for withdrawers at the expense of people that are staying in the vault, but I think it's reasonable to do and keeps us closer to the "best case" state of having the correct ratio of allocation.

Reply:
Cool. I put it towards the end of the [Everlong task list](https://github.com/delvtech/hyperdrive/issues/1044) since it's not "mission critical".